diff --git a/node_modules/buffer-to-data-url/index.d.ts b/node_modules/buffer-to-data-url/index.d.ts
index 109ee73..9e47b41 100644
--- a/node_modules/buffer-to-data-url/index.d.ts
+++ b/node_modules/buffer-to-data-url/index.d.ts
@@ -1,10 +1,3 @@
-/** @module buffer-to-data-url
- */
-declare module "buffer-to-data-url" {
-    /**
-     * @param {string} mimeType
-     * @param {Buffer} buffer
-     * @return {Promise<string>}
-     */
-    export default function(mimeType: string, buffer: Buffer): Promise<string>;
-}
\ No newline at end of file
+declare function bufferToDataUrl(mimeType: string, buffer: Buffer): Promise<string>;
+
+export = bufferToDataUrl;
